src/ngraph_builder.cc

```diff
@@ -1273,6 +1273,67 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
       // Do nothing! NoOps sometimes get placed on nGraph for bureaucratic
       // reasons, but they have no data flow inputs or outputs.
     }
+    // -------
+    // Pack
+    // -------
+    else if (op->type_string() == "Pack") {
+
+      std::cout << "Number of inputs : " << op->num_inputs() << std::endl;
+      if (op->num_inputs() < 1) {
+        return tf::errors::InvalidArgument(
+            "Number of inputs should be at least 1 for Pack");
+      }
+
+      ng::NodeVector ng_concat_inputs;
+
+      for (size_t i = 0;i < op->num_inputs();++i) {  
+        tf::Node* tf_input;
+        TF_RETURN_IF_ERROR(op->input_node(i, &tf_input));
+        auto ng_input = ng_op_map.at(tf_input->name());
```

Comment:
Need a try catch here. The lookup will throw exception - you've done this before - so you know what to do :)